packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Details/cards/ChartsCard.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react';
+import { DateTime } from 'luxon';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { MigrationModelGroupVersionKind, V1beta1Migration } from '@kubev2v/types';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartTooltip } from '@patternfly/react-charts';
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  Dropdown,
+  DropdownItem,
+  Flex,
+  FlexItem,
+  KebabToggle,
+} from '@patternfly/react-core';
+
+import { MigrationsCardProps } from './MigrationsCard';
+
+interface MigrationDataPoint {
+  dateLabel: string;
+  value: number;
+}
+
+const toStarted = (m: V1beta1Migration): string => m.status.started;
+const toFinished = (m: V1beta1Migration): string => m.status.completed;
+const hasTimestamp = (timestamp: string) => timestamp && DateTime.fromISO(timestamp).isValid;
+const toDateTime = (timestamp: string): DateTime => DateTime.fromISO(timestamp);
+const isLastWeek = (date: DateTime) => date.diffNow('days').get('days') <= 7;
```

Comment:
using the word "week" is problematic, lets use "last7Days" and "last24Hours" or some other phrase to be more clear we don't mean weeks and days but, N days and M hours.

a. we show last 7 days, may not be a complete one week, can be some of this week and some of another
b. we may want to change the time to 10 days instead of 7 ...